mongodbatlas/description/database-user.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package description
+
+const (
+	DatabaseUserResource = "`mongodbatlas_database_user` provides a Database User resource. This represents a database user which will be applied to all clusters within the project.\n\nEach user has a set of roles that provide access to the project’s databases. User's roles apply to all the clusters in the project: if two clusters have a `products` database and a user has a role granting `read` access on the products database, the user has that access on both clusters."
```

Comment:
word of advise, a single "description" package becomes hard to maintain, for example the const `ProjectID` has db user specific copy which tends risk somebody copy pasting the `ProjectID` const without realizing and havign then the wrong copy

unless the descriptions will be reused there's no benefit defining a const over in-lining the value (the compiler will actually do that, inline the value where defined) so consider this on how to move forward with the descriptions

Reply:
I removed the description